models/issue_comment.go

```diff
@@ -108,6 +108,8 @@ const (
 	CommentTypeDismissReview
 	// 33 Change issue ref
 	CommentTypeChangeIssueRef
+	// 34 Change confidential
+	CommentTypeConfidentialChanged
```

Comment:
Needs an update of string list below
https://github.com/go-gitea/gitea/blob/c560ddbdf5a5ce6cf5f459b19662b08a667fc244/models/issue_comment.go#L149